lambdas/data-migration2/README.md

```diff
@@ -1,14 +1,17 @@
 # Data Migration 2 Lambda
 
-## The second of two Lambdas to be run in order to migrate existing DynamoDB data to PostgreSQL.
+The data-migration2 lambda is the second of two Lambdas to be run in order to migrate existing DynamoDB data to PostgreSQL.
 
 ### Running the API
-This second Lambda in the data migration process can be run in two ways:
+This second Lambda in the data migration process can be run in several ways:
 
 1. In an ECS container using the Cumulus ECS Service
 2. Directly from the AWS console / command line
+3. By invoking an async operation using the provided `${PREFIX}-postgres-migration-async-operation` lambda and invoking from the AWS console or command line.
```

Comment:
Option 3 is just a repeat of option 2 with more detail. And option 1 is no longer valid. We should just make it clear that there is only one way to run this migration.